wagtail/admin/wagtail_hooks.py

```diff
@@ -906,7 +906,10 @@ def is_shown(self, request):
 
 class WorkflowReportMenuItem(MenuItem):
     def is_shown(self, request):
-        return getattr(settings, "WAGTAIL_WORKFLOW_ENABLED", True)
+        return (
+            getattr(settings, "WAGTAIL_WORKFLOW_ENABLED", True)
+            and UserPagePermissionsProxy(request.user).can_publish_pages()
```

Comment:
Is this correct? I would have thought that an ordinary editor, who doesn't have permission to publish pages themselves but can submit them to a workflow, would be one of the user types who would benefit the most from seeing this report.

Reply:
First of all, sorry for not responding for so long @gasman I somehow missed your comment.

Yeah, that's a fair point. Editor should be able to have (read only) access to pages he sent for moderation. I so I think solution would be going with `UserPagePermissionsProxy(request.user).can_edit_pages()` instead of `UserPagePermissionsProxy(request.user).can_publish_pages()` in case of `WorkflowReportMenuItem.is_shown()`, so limited users still can't see workflows, but editors can.